github-actions/prepare-mysql/README.md

```diff
@@ -0,0 +1,30 @@
+# Prepare MySQL in GitHub Actions
+
+This action provides the following functionality for GitHub Actions users:
+
+- Enable MySQL service
+- Handle the authentication compatibility for PHP 7.0-7.3
+- Log the version information
+
+## Usage
+
+See [action.yml](action.yml)
+
+#### Basic:
+
+```yaml
+jobs:
+  UnitTests:
+    name: Unit tests with database
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Prepare MySQL
+        uses: woocommerce/grow/github-actions/prepare-mysql@github-actions
```

Comment:
Maybe it would be safer to use commit hash, to avoid problems like https://github.com/woocommerce/google-listings-and-ads/pull/1552, especially that this branch may get removed after this PR is merged.

However, branch names are much easier to read :confused: 

Seems we need to solve https://github.com/woocommerce/grow/issues/5 sooner than later

Reply:
Phases 4 and 5 in pb0Spc-2DL-p2#comment-5753 are creating the versioning management for the GitHub actions in this repo. With the versioning feature, the caller repos will use `@v1-actions` to apply actions instead of a specific commit or a branch.

The `@v1-actions` will keep pointing to the latest released `v1.x.x-actions` tag, Thus, caller repos will always have the latest actions by using the `@v1-actions` major version.

I have completed the development of versioning feature in the `add/versioning-action` branch and tested it with a demo release in another caller repo. In the PR 139-gh-woocommerce/automatewoo-referrals, we can find that the uses of `uses: woocommerce/grow/github-actions/prepare-php@v1-actions-pre` work well.

All the uses in all caller repos will be changed to `uses: woocommerce/grow/github-actions/<action-directory>@v1-actions` once the feature is formally released. I will create a separate PR later.